scss/nav-help.scss

```diff
@@ -40,3 +40,41 @@ margin-bottom:0.5em;
         flex-direction:column;
         justify-content:space-around;
     }
+    .nav-help-mobile{
+        z-index:1;
+        position: absolute;
+        background: #0003;
+        color:white;
+        font-family:"Markazi",serif;
+        font-size:10px;
+        padding:0 1% 0 1%;
+        overflow:hidden;
+        bottom: 48px;
+        margin: 2%;
+        border: solid #22224433 2px;
+        display:flex;
```

Comment:
```suggestion
```
`display:flex` is conflicting with `display:none` for desktop and can be removed.